internal/controller/pkg/manager/reconciler.go

```diff
@@ -284,10 +284,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 
 	revisionName, err := r.pkg.Revision(ctx, p)
 	if err != nil {
-		log.Debug(errUnpack, "error", err)
 		err = errors.Wrap(err, errUnpack)
+		p.SetConditions(v1.Unpacking())
 		r.record.Event(p, event.Warning(reasonUnpack, err))
-		return reconcile.Result{}, err
+		return reconcile.Result{RequeueAfter: time.Minute}, errors.Wrap(r.client.Status().Update(ctx, p), errUpdateStatus)
```

Comment:
We previously opted to return the error to enqueue immediately and rely on exponential backoff here. I'm open to adjusting, but 1 minute feels like it could be a fairly long time to wait just to attempt fetching the digest again. Do you think updating to always enqueue after 1 minute makes more sense?

Reply:
@sttts it depends -- I believe we have used it in places where the issue is likely to be transient or requiring some external intervention (in this case we likely either have a temporary network issue, or the digest does not exist at all). In those cases we want to retry quickly, then retry infrequently, as opposed to retrying on a fixed interval (e.g. 1 minute). Worth noting that we globally configure backoff today, so there is likely some room for improvement.

Reply:
> Do you think updating to always enqueue after 1 minute makes more sense?

Not exactly sure what you mean here. Some agressive backoff with 1min max would IMO make most sense. Wdyt?

Reply:
Check again. That's what you mean?

Reply:
Here is the logic in controller-runtime (fun-fact: one must go into the implementation to find the semantics): https://github.com/kubernetes-sigs/controller-runtime/blob/451530c09937350ff7f5ebf7a00d059b4334e8ee/pkg/internal/controller/controller.go#L325

Reply:
> I think just returning the error is fine for now here.

:+1:

Reply:
> Here is the logic in controller-runtime (fun-fact: one must go into the implementation to find the semantics):

Also linked in the post :)